angular/playground/projects/ng-forms-manager/src/lib/forms-manager.ts

```diff
@@ -137,9 +138,22 @@ export class AkitaNgFormsManager<FormsState = any> {
       }
     }
 
-    this.valueChanges[formName as any] = merge(form.valueChanges, form.statusChanges.pipe(distinctUntilChanged()))
-      .pipe(debounceTime(merged.debounceTime))
-      .subscribe(() => this.updateStore(formName, form));
+    const formChanges$ = merge(form.valueChanges, form.statusChanges.pipe(distinctUntilChanged())).pipe(
```

Comment:
You don't need to touch this method. As I said, the change should be made in the `unsubscribe` method.

Reply:
Simply misunderstood, will update + add injection token + ping back here when its done.